setuptools/dist.py

```diff
@@ -289,6 +289,10 @@ def assert_bool(dist, attr, value):
         raise DistutilsSetupError(tmpl.format(attr=attr, value=value))
 
 
+def invalid(dist, attr, value):
+    raise DistutilsSetupError(f"{attr} is invalid.")
```

Comment:
@jaraco in the light of https://github.com/pypa/setuptools/issues/2775, maybe this could at least be turned into
```suggestion
    if not value:  # it's okay, if it's False
        warnings.warn(f"Remove the {attr} setting from your packaging.")
        return
    raise DistutilsSetupError(f"{attr} is invalid.")
```

Reply:
I originally was writing the function with this check, but then I wanted to make the "check" function suitable for any invalid value. I'll review the referenced bug and consider this proposal.

Reply:
Yeah, it turned out more impactful on the scale than you've probably expected. Seems rather urgent.